doc/reference/testing.rst

```diff
@@ -526,6 +526,93 @@ Two new command line arguments were added since Odoo 13.0 to control this behavi
 :option:`--screenshots <odoo-bin --screenshots>` and :option:`--screencasts <odoo-bin --screencasts>`
 
 
+Performance Testing
+===================
+
+Query counts
+------------
+
+One of the ways to test performance is to measure database queries.
+Manually, this can be tested with the `--log-sql` CLI parameter.
+If you want to fix the maximum number of queries for an operation,
+you can use the :meth:`~odoo.tests.common.BaseCase.assertQueryCount` method,
+integrated in Odoo test classes.
+
+.. code-block:: python
+
+    with self.assertQueryCount(11):
+        do_something()
+
+.. _reference/testing/populate:
+
+Database population
+-------------------
+
+Odoo CLI offers a :ref:`database population<reference/cmdline/populate>` feature.
+
+.. code-block:: console
+
+    odoo-bin populate
+
+Instead of the tedious manual, or programmatic, specification of test data,
+one can use this feature to fill a database on demand with the desired number of test data.
+This can be used to detect diverse bugs or performance issues in tested flows.
+
+To enable this feature for a given model, the following methods|attributes must be defined.
+They all have a default in the Base model, but you have to define at least the method
```

Comment:
```suggestion
They all have a default in ``BaseModel``, but you have to define at least the method
```
Not sure about this... But I find that "the Base model" doesn't immediately refer to odoo.models.BaseModel

Reply:
Why not indicating the default directly in the parameters instead of this sentence (and required for `_populate_factories`)?